crates/typst/src/introspection/counter.rs

```diff
@@ -405,14 +405,13 @@ impl Counter {
         ///
         /// - If it is a string, creates a custom counter that is only affected
         ///   by manual updates,
-        /// - If this is a `{<label>}`, counts through all elements with that
-        ///   label,
-        /// - If this is an element function or selector, counts through its
-        ///   elements,
-        /// - If this is the [`page`] function, counts through pages.
+        /// - If it is the [`page`] function, counts through pages,
+        /// - If it is a `{<label>}`, counts through all elements with that label,
+        /// - If it is an element function or other [selectors]($selector), counts
```

Comment:
selector should be in singular here. I would also move this above the label case because the label case is pretty rare.

Reply:
Understood. I wanted to change this comment block, because technically element functions and labels are all [selectors](https://typst.app/docs/reference/foundations/selector/), so the wording had better _not_ imply that selectors are totally unrelated to element functions or labels.

Therefore, the new commit reworded it further. If it's not to your liking, feel free to modify directly :)